machine_test.go

```diff
@@ -141,8 +141,8 @@ func TestJailerMicroVMExecution(t *testing.T) {
 	os.MkdirAll(jailerTestPath, 0777)
 
 	socketPath := filepath.Join(jailerTestPath, "firecracker", "TestJailerMicroVMExecution.socket")
-	logFifo := filepath.Join(testDataPath, "firecracker.log")
-	metricsFifo := filepath.Join(testDataPath, "firecracker-metrics")
+	logFifo := filepath.Join(tmpDir, "firecracker.log")
```

Comment:
How come this was changed?

Reply:
the `testDataPath` should only house read only files, as this will create issues across tests, more specifically this one due to linking failing since `firecracker.log` or `firecracker-metrics` potentially existing.